packages/cipherstash-proxy/src/error.rs

```diff
@@ -6,7 +6,7 @@ use std::io;
 use thiserror::Error;
 use tokio::time::error::Elapsed;
 
-const ERROR_DOC_BASE_URL: &str = "https://github.com/cipherstash/proxy/docs/errors.md";
+const ERROR_DOC_BASE_URL: &str = "https://github.com/cipherstash/proxy/blob/main/docs/errors.md";
```

Comment:
Without `blob/main`, it's not found

Reply:
Oh. That's a bit annoying.

Reply:
Ideally, we should ascertain the current version of the proxy and link to a specific commit so that the release and the docs we link to are in sync.

However, that would require some proc_macro shenanigans to interpolate the appropriate tag into the URL so I would be happy to shave that yak later.